### ISSUE TITLE
Running gitex.cmd from powershell doesn't ever load log panel. 

### ISSUE THREAD

**Issue by finlaybob** [gitextensions/gitextensions#5629]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
When running
`& "C:\Program Files (x86)\GitExtensions\gitex.cmd"` 
from powershell inside a git directory, the program starts but the log panel never shows up - it's just spinning dots. 
![image](https://user-images.githubusercontent.com/2033345/47566642-3f5e6100-d924-11e8-8029-44fc61facc92.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Open a powershell prompt
1. run  `& "C:\Program Files (x86)\GitExtensions\gitex.cmd"`
1. Observe spinning dots for as long as you like. 
1. Press F5 to refresh and the log will show. 

**What is the expected behavior?**
Should not have to press F5 to display the log.  

**Environment you encounter the issue:**
- GitExtensions version: 3.00.00.01-beta1
- GIT version:  2.19.0.windows.1
- OS version: Windows 10 Pro 1803 (Build 17134.345)
- .NET version: 4.7.1

**Did this work in previous version of GitExtensions (which)?**
Yes. Most recently 2.51.05

**Comment by RussKie** [comment]
Can you post your gitcommand log (it is under menu _Tools_)?

**Comment by finlaybob** [comment]
Well... heres another bug 🤣: This is the log saved to a file:
```
GitCommands.Logging.CommandLogEntry
GitCommands.Logging.CommandLogEntry
GitCommands.Logging.CommandLogEntry
GitCommands.Logging.CommandLogEntry
GitCommands.Logging.CommandLogEntry
GitCommands.Logging.CommandLogEntry
GitCommands.Logging.CommandLogEntry
```
[This is an image of the log instead](https://i.imgur.com/dF29SCs.png)

**Comment by finlaybob** [comment]
[Here is the log after pressing F5](https://i.imgur.com/LZOdPM9.png)

**Comment by vbjay** [comment]
You can paste images straight into comments. No need for external site.

**Comment by finlaybob** [comment]
> If you press F5 or click any of the toolbar icons, does anything happen?

Yes I mentioned F5 works. Also clicking something that would refresh after - e.g. checking out a different branch - makes it refresh and then the log shows up. 

> If you don't launch from PowerShell, does it work?

Yes works fine. It opens to the dashboard, this then requires opening the repo, which is forcing a refresh once it's loaded.  


Additionally
- Running with the `browse`  commandline option means it works correctly. e.g.  `& "C:\Program Files (x86)\GitExtensions\gitex.cmd" browse` works

**Comment by drewnoakes** [comment]
> I mentioned F5 works

Sorry, I missed that.

Thanks for the extra info. Hopefully this repros on other machines.

**Comment by RussKie** [comment]
I attached a debugger and it looks like the background event never arrived, and we're stuck an infinite loop waiting for it....

![image](https://user-images.githubusercontent.com/4403806/47830624-cf971d00-dde0-11e8-8789-8238d764bd8d.png)

**Comment by spdr870** [comment]
> If I run a debug version of the app from `<path>\gitextensions\GitExtensions\bin\Debug` it gets stuck at the loading screen and I have to manually refresh it.
> There is nothing in a gitcommand log to indicate something is failing
> 
> @spdr870 could you please try it on your machine?

I think I fixed this in #5722 . Didn't know about this issue.
